lisa/tests/staging/sched_android.py

```diff
@@ -0,0 +1,290 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2019, Arm Limited and contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+import os.path
+import abc
+
+from lisa.wlgen.rta import RTA, Periodic
+from lisa.tests.base import ResultBundle, CannotCreateError, RTATestBundle
+from lisa.trace import Trace, FtraceCollector, FtraceConf, requires_events
+from lisa.target import Target
+from lisa.utils import ArtifactPath
+
+class SchedTuneTest(RTATestBundle, abc.ABC):
```

Comment:
@douglas-raillard-arm I'm fairly sure I already asked this in the past, but `RTATestBundle` already being abstract, do we need to re-inherit from `abc.ABC`?

Reply:
Yeah I was wondering that too, but that's what we do for `EASBehaviour` so I though I should just do the same here

Reply:
We don't do that for `MisfitMigrationBase` so I think that's superfluous.

Reply:
Since RTATestBundle already inherits from it, it is superfluous. It's only useful as a remainder "that class contains abstractmethods", which will remain abstract even if one day we decide RTATestBundle is not an ABC anymore (otherwise, it could silently become non-abstract).